src/applications/personalization/dashboard/components/cerner-widgets.js

```diff
@@ -0,0 +1,128 @@
+import React, { Fragment } from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import environment from 'platform/utilities/environment';
+import { mhvUrl } from 'platform/site-wide/mhv/utilities';
+
+// Returns an AlertBox to present the user with info about working with the
+// Cerner facility they are enrolled at. Props allow you to edit a small amount
+// of the content that is rendered in the AlertBox.
+const CernerAlertBox = ({
+  ctaButtonText,
+  ctaButtonUrl,
+  ctaText,
+  facilityNames,
+}) => {
+  // Helper component that takes an array of facility names and a separator string and returns some JSX to style the list of facility names.
+  const FacilityList = ({ facilities, separator }) => {
+    const newArray = [];
+    // first make an array that alternates between a facility name and the
+    // separator word wrapped in a <span> for styling purposes
+    facilities.forEach(el => {
+      newArray.push(el);
+      newArray.push(
+        <span className="vads-u-font-weight--normal vads-u-font-size--base">
```

Comment:
Instead of using `push` for the separator, what do you think of using `join` so we don't need to `pop` the last item? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
I might be overlooking something, but could we replace this whole block with `facilities.join(<span className="vads-u-font-weight--normal vads-u-font-size--base">{separator}</span>`? Not sure if `join` takes in HTML elements, if not, maybe we can add a className to the span? We then wouldn't need to `map` over the `newArray`? 

Or did you specifically go the array approach because `join` does not take `span`s?

Reply:
I actually tried that at first. And ended up with that `<span>` markup being rendered in the DOM.